etl/src/main/java/org/pih/hivmigration/etl/sql/AccompagnateurMigrator.groovy

```diff
@@ -0,0 +1,39 @@
+package org.pih.hivmigration.etl.sql
+
+class AccompagnateurMigrator extends ObsMigrator {
+
+    @Override
+    def void migrate() {
+
+        create_tmp_obs_table()
+
+        executeMysql("Load Accompagnateur names as observations on the most recent followup or intake form", ''' 
+                
+        INSERT INTO tmp_obs (
+            value_text, 
+            source_patient_id, 
```

Comment:
`source_patient_id` is no longer used. It's inferred from `source_encounter_id`.

Reply:
Thanks! I will remove it.